custom_components/babybuddy/services.yaml

```diff
@@ -1,6 +1,6 @@
-services_children_add:
+add_child:
```

Comment:
In order to keep this (mostly) backwards compatible, please keep service IDs the same (e.g., `services_children_add` vs. `add_child`).

Reply:
Giving that this a major update and is considered as a breaking change a user would be expected to update his scripts and automations to reflect these updates. If you want to focus on backward compatibility then all the service names and fields should be looked into. What do you think.

Reply:
I'm sorry, for some reason I thought you only bumped as minor. These can be left as is.